examples_tests/31.SkinningDataBenchmark/2.vert

```diff
@@ -0,0 +1,26 @@
+#version 460 core
+
+layout( push_constant, row_major ) uniform Block 
+{
+	mat4 vp;
+}pushConstants;
+
+layout(std430, set = 0, binding = 0, row_major) restrict readonly buffer BoneMatrices
+{
+    mat4x3 matrix[];
+};
+
+layout(location = 0) in vec3 pos;
+layout(location = 3) in vec3 normal;
+layout(location = 4) in int boneID;
+
+layout(location = 1) out vec3 worldPos;
+layout(location = 2) out vec3 vNormal;
+
+void main()
+{
+    vec3 vWorldPos = matrix[boneID] * vec4(pos, 1.0);
+    gl_Position = pushConstants.vp * vec4(vWorldPos, 1.0);

```

Comment:
NO, this will skew benchmark results, its also horribly inefficient.

The ViewProj matrix should have been already concatenated with the bone's world matrix in the routine that updates bone matrices (be it a GPU Compute shader or CPU).

Reply:
so no need for the VP in the push constants